src/app/about-us/about-us.component.html

```diff
@@ -1,3 +1,28 @@
-<p>
-  about-us works!
-</p>
+<div class="container">
```

Comment:
I don't think you need to have this div with the container class, since the content is aligned to the center.